app/(pages)/programs/protege-program/page.tsx

```diff
@@ -0,0 +1,60 @@
+import {Box, Stack, AspectRatio} from '@mui/joy';
+import PageHeaderCard from "@/components/PageHeaderCard";
+import Image from 'next/image';
+import HeadingBodyText from '@/components/HeadingBodyText';
+import ProgramsTitleDescription from '@/components/ProgramsTitleDescription';
+
+export default function ProtegeProgram() {
+
+  const bodyText = 'The Protege Program is a mentoring program where female-identifying first-year engineering students will be grouped with a mentor studying engineering in their second year or higher to learn more about UNSW and engineering at UNSW. It is a great opportunity to make friends with other engineering girls and gain some useful advice from older students.'
+
+  const title1 = 'Meet other female engineers in your field'
+  const description1 = 'The Protege Program offers the chance for new female engineering students to meet and build a network with other female engineering students in first year and beyond through the course of a structured program.'
+
+  const title2 = 'Foster friendships'
+  const description2 = 'The Protege Program offers the chance for new female engineering students to meet and build a network with other female engineering students in first year and beyond through the course of a structured program.'
+
+  const title3 = 'Ease the transition'
+  const description3 = 'The Protege Program helps ease the transition between high school and engineering at UNSW by providing an older student mentor who can help to navigate this new environment and overcome its challenges.'
+
+
+  return (
+
+    <Box>
+      <PageHeaderCard imagePath={'/programs/header.jpg'} pageTitle={'Programs'}/>
+      <Box sx={{ px: '30px' }}>
```

Comment:
the headingbodytext doesn't need to be in a box, to keep it consistent with the other pages